tests/integration_tests/vectorstores/test_chroma.py

```diff
@@ -39,7 +39,47 @@ def test_chroma_with_metadatas_with_scores() -> None:
         metadatas=metadatas,
     )
     output = docsearch.similarity_search_with_score("foo", k=1)
-    assert output == [(Document(page_content="foo", metadata={"page": "0"}), 1.0)]
+    assert output == [(Document(page_content="foo", metadata={"page": "0"}), 0.0)]
```

Comment:
I'm not exactly sure how this test was passing with an expectation of distance `1.0` before. By my reading the query embeddings should always be an exact match to the first element of `texts` and always return 0 distance for that result